### ISSUE TITLE
Documentation search is broken with new versioning mode

### ISSUE THREAD

**Issue by joshrosso** [vmware-tanzu/community-edition#3653]
I believe due to the new versioning updates, our search functionality has broken.

As you can see in the URL bar after the search, the path resolves as `$VERSION/latest/$RESOURCE` when it should be `$VERSION/$RESOURCE`.

![image](https://user-images.githubusercontent.com/6200057/159497119-8e845ba6-a18f-4f6f-a4b1-fb26522b66b4.png)

![image](https://user-images.githubusercontent.com/6200057/159496984-3587e75c-b260-4b66-be29-ef9fb9a8e897.png)

**Comment by stmcginnis** [comment]
@garrying was that you?

I can start looking into it, but will need to get up to speed on how it works.

**Comment by jpmcb** [comment]
Correct: it was done in https://github.com/vmware-tanzu/community-edition/issues/1243

/assign @garrying
